utils/anchor.js

```diff
@@ -12,20 +12,27 @@ const anchor = (md, options) => {
     const slug = slugify(contentToken.content);
 
     if (tokens[index].tag === 'h1') {
-      return `<${tokens[index].tag} id="${slug}" class="mb-400">`;
+      return `<${tokens[index].tag} id="${slug}" class="mt-0">`;
     } else if (tokens[index].tag === 'h2') {
-      return `<${tokens[index].tag} id="${slug}" class="mt-500 mb-400">`;
+      return `<${tokens[index].tag} id="${slug}" class="mt-500">`;
     } else if (tokens[index].tag === 'h3') {
-      return `<${tokens[index].tag} id="${slug}" class="mt-500 mb-400">`;
+      return `<${tokens[index].tag} id="${slug}" class="mt-500">`;
     } else if (tokens[index].tag === 'h4') {
-      return `<${tokens[index].tag} id="${slug}" class="mt-500 mb-400">`;
+      return `<${tokens[index].tag} id="${slug}" class="mt-500">`;
     } else {
       return `<${tokens[index].tag} id="${slug}">`;
     }
   };
 
+  md.renderer.rules.ordered_list_open = function (tokens, index) {
+    return `<${tokens[index].tag} class="list-decimal mt-400">`;
+  };
+  md.renderer.rules.bullet_list_open = function (tokens, index) {
+    return `<${tokens[index].tag} class="list-disc mt-400">`;
+  };
+
   md.renderer.rules.paragraph_open = function(tokens, index) {
-    return `<${tokens[index].tag} class="mb-400">`;
+    return `<${tokens[index].tag} class="mt-400">`;
```

Comment:
Using top margin might be easier to work with. Margin top works magic with the `+` css combinator. I've used it to keep paragraphs and lists closer together.

Reply:
Please change this all back to use bottom margin. As far as I'm aware, it is general best practice in design to apply bottom margin to elements within unkown content than top margin so I don't understand why we need to change this to something that is less predictive.

Reply:
reverted

Reply:
Thank you!